compiler/quilt/tools/build.py

```diff
@@ -106,9 +113,9 @@ def _gen_glob_data(dir, pattern, child_table):
         node_table = {} if child_table is None else child_table.copy()
         filepath = filepath.relative_to(dir)
         node_table[RESERVED['file']] = str(filepath)
-        node_name = to_nodename(filepath.stem, invalid=used_names)
+        node_name = _to_nodename(filepath.stem, invalid=used_names)
```

Comment:
Can we avoid renaming functions in a PR like this? It'll be much easier to follow the diffs if small, but widespread changes like that happen in their own PRs.